### ISSUE TITLE
Player controls are not accessible with screen readers on Firefox  

### ISSUE THREAD

**Issue by ai25** [vidstack/player#892]
### Current Behavior:
When using a screen reader:
1. Buttons are not being activated using double-tap.
2. Player is untargetable when controls are hidden.

### Expected Behavior:
1. Buttons should be activated with double-tap.
2. Player should be targetable and activating it should show the controls.
### Steps To Reproduce:
1. Activate SR.
2. Open Firefox on Android
3. Interact with player.

### Environment:
- Framework: any
- Meta Framework: any
- Node: any
- Device: Google Pixel 6 Pro
- OS: Android 13
- Browser: Firefox 112.2.0

### Anything Else?
Issue is only reproducible on Firefox, Chrome works as expected.
[Video](https://files.catbox.moe/hwai86.mp4)

**Comment by julezrulez** [comment]
Same kind of problem. Focus with keyboard works fine, activating buttons however does not work.
- OS: Windows 11
- Device: Dell laptop
- Browser: Firefox 116.0.2
- Assistive software: NVDA
Chrome works as expected.

**Comment by mihar-22** [comment]
Thanks for reporting! I'll look into this right after our next release, after we confirm it's still happening.

**Comment by mihar-22** [comment]
Sorry for the late response, this should've been resolved a while ago. If any issues persistslet me know and we can reopen.
